packages/utils/tokens/package.json

```diff
@@ -32,13 +32,25 @@
   },
   "devDependencies": {
     "@fluentui-react-native/eslint-config-rules": "^0.1.1",
-    "@types/react-native": "^0.66.0",
     "@fluentui-react-native/scripts": "^0.1.1",
-    "react-native": "^0.66.0"
+    "@types/react-native": "^0.66.0",
```

Comment:
The types package should be bumped to 0.68 as well. The numbers of the types package generally match the RN version number they support.

Reply:
Thanks for the callout! I'll go update that.

Reply:
@kelset is this something rnx-dep-check should have done but missed?